docs/index.rst

```diff
@@ -9,8 +9,53 @@ Pangeo
 .. image:: _static/pangeo_simple_logo.svg
    :width: 175 px
 
-A community platform for Big Data geoscience
-============================================
+A community promoting open, reproducible, and scalable geoscience
+=================================================================
+
+.. list-table:: 
+   :widths: 10 10 10 10 10 
+   :header-rows: 1
+   :align: left
+
+   * - ❓ Have questions about Python software?
+     - 🤩 Want to hear about new updates to Pangeo software or infrastructure?
+     - 😃 Wondering how you can get involved in the Pangeo community?
+     - ☁️ Interested in getting access to Pangeo Cloud?
+     - 🧐 Just curious to learn more about Pangeo?
+   * - Search for answers or ask your own question on the `Pangeo Discourse Forum <https://discourse.pangeo.io/>`_!
+     - Check out our `Medium blog <https://medium.com/pangeo>`_ and our `Twitter feed <https://twitter.com/pangeo_data>`_!
+     - Post or answer questions in the `Pangeo Discourse Forum <https://discourse.pangeo.io/>`_, attend a `community meeting <https://pangeo.io/meeting-notes.html>`_, or start issues/create pull requests on our `GitHub repos <https://github.com/pangeo-data/>`_!
+     - Fill out the application form available on the `Pangeo Cloud site <https://pangeo.io/cloud.html>`_.
+     - Read about us on the `About Pangeo page <https://pangeo.io/about.html>`_.
+
+..
+   This portion is currently commented out rather than deleted, because it appears that the "toctree" here may be 
```

Comment:
Yep, this is what sphinx uses to figure out which documents in the build (https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=toctree#table-of-contents).

Did you comment it out since you didn't want to include the toctree on the homepage? If you want, you can do something like

```
.. toctree::
   :hidden:

   about
   ...
```


to hide the toctree. I think we'd still want some way to navigate to those pages though, and I'm not sure how `hidden` interacts with the sphinx's navigation stuff.